src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/product-part-list.tpl.html

```diff
@@ -0,0 +1,89 @@
+<div class="blade-static">
+    <div class="form-group">
+        <div class="form-input __search">
+            <input ng-model="filter.keyword"
+                   placeholder="{{'platform.placeholders.search-keyword' | translate}}"
+                   ng-keyup="$event.which === 13 && filter.criteriaChanged()" />
+            <button class="btn __other"
+                    style="position: relative; right: 45px;">
+                <i class="btn-ico fa fa-times-circle"
+                   title="Clear"
+                   ng-click="filter.keyword=null;filter.criteriaChanged()"
+                   aria-hidden="true">
+                </i>
+            </button>
+        </div>
+        <p class="form-count">{{ 'platform.list.count' | translate }}: <span class="count">{{pageSettings.totalItems | number:0}}</span></p>
+    </div>
+</div>
+<div class="blade-static __bottom" ng-if="pageSettings.itemsPerPageCount < pageSettings.totalItems" ng-include="'pagerTemplate.html'"></div>
+<div class="blade-content">
+    <div class="blade-inner">
+        <div class="table-wrapper" ng-init="setGridOptions({
```

Comment:
Can we move it to the controller?

Reply:
We can, but I will suggest leave as is same way it specified in all platform modules: it contains grid display options, column sizing, row template names, display names through localization resources and so on, so it more helpful for developer when it live here.